### DISCUSSION TITLE
[Mod proposal] Hitting a later circle and missing all circles before it

### DISCUSSION THREAD

**Discussion by superpowers04** [ppy/osu#30700] [Ideas]
Circles usually have to be hit in order, however I think it would be an interesting mod to allow out of order circle hits or allow out of order circle hits and missing every circle before the circle you actually hit.

**Comment by N0Y0U2114** [comment]
How would this work timing wise? This is already what (kinda) happens in lazer since it doesnt have note lock, you get the off timing for the circle you hit (if close enough it its timing), and you miss the rest. Not quite sure what you have in mind
